node_lambnik/docker-compose.yml

```diff
@@ -14,8 +14,6 @@ services:
     image: quay.io/azavea/postgis:2.3-postgres9.6-slim
     expose:
       - "5432"
-    volumes:
-      - /opt/dbdata/lambnik:/var/lib/postgresql/data
```

Comment:
I think the reason this is normally included is to persist the database between the cycles of `docker-compose` up/down.  I'd verify that this change still works for you normal dev workflow.

Reply:
I suggested this because we were seeing differences between our ostensibly identical dev instances. The database here is basically a test fixture, so I think making it more ephemeral makes sense. Persisting it on the host seems to me like it adds more potential gotchas than benefits.

Reply:
My stance on having that volume is that it's more of a hassle to have that data persisted and then be confused when you forget about it and need freshly create the environment than it is to have to run `update` more often, since the `update` --> `server` process is already specified in the README but it might not be totally clear that the database info is being persisted locally.